series/praha-pyvo/events/2020-10-21.yaml

```diff
@@ -0,0 +1,36 @@
+city: praha
+start: 2020-10-21 19:00:00
+name: Pražské Pyvo
+number: 115
+venue: tba
```

Comment:
```suggestion
```

Reply:
jop, taky uz mi doslo, ze jsem to tam mel nedavat :D ale zase jsem mel radost, ze jsem to zprovoznil :D :D :D